docs/api/index.md

```diff
@@ -3,15 +3,12 @@
 The following types are used in the type signatures below
 
 ```ts
-type DoneCallback = (error?: any) => void
 type Awaitable<T> = T | PromiseLike<T>
-type TestFunction = () => Awaitable<void> | ((done: DoneCallback) => void)
+type TestFunction = () => Awaitable<void>
 ```
 
 When a test function returns a promise, the runner will wait until it is resolved to collect async expectations. If the promise is rejected, the test will fail.
 
```

Comment:
Maybe we could leave a tip here pointing to the migration guide? At least for a few months, I don't think it would be useful for it to remain forever.

```suggestion

::: tip 
In Jest, `TestFunction` can also be of type `(done: DoneCallback) => void`. If this form is used, the test will not be concluded until `done` is called. You can achieve the same using an `async` function, see the [Migration guide Done Callback section](../guide/migration#done-callback).
:::

```